autoload/sy/repo.vim

```diff
@@ -629,7 +634,8 @@ let s:default_vcs_cmds = {
       \ 'rcs':      'rcsdiff -U0 %f 2>%n',
       \ 'accurev':  'accurev diff %f -- -U0',
       \ 'perforce': 'p4 info '. sy#util#shell_redirect('%n') . (has('win32') ? ' &&' : ' && env P4DIFF= P4COLORS=') .' p4 diff -du0 %f',
-      \ 'tfs':      'tf diff -version:W -noprompt -format:Unified %f'
+      \ 'tfs':      'tf diff -version:W -noprompt -format:Unified %f',
+      \ 'jj':       'jj diff --color=never --context=0 -r @ -- %f',
```

Comment:
This should have a `--git` as well, right?

Reply:
yep good call. fixed